charts/kong/values.yaml

```diff
@@ -403,6 +403,9 @@ podDisruptionBudget:
 
 podSecurityPolicy:
   enabled: false
+  # Make the root filesystem read-only. This is not compatible with Kong Enterprise <1.5.
+  # If you use Kong Enterprise <1.5, this must be set to false.
+  readOnlyRootFilesystem: true
```

Comment:
Agree that we should hold this until 1.5 is out, but are we concerned with having it be the default then? It still only applies to users that have specifically enabled PSPs.

Worst case users already have it enabled and don't read the upgrade guide. Their GUI will be broken because it can't write its config, while the proxy and admin API will function normally, so fairly minor impact.